src/utils/url-utils.ts

```diff
@@ -0,0 +1,102 @@
+import { useHistory } from 'react-router-dom';
+import moment from 'moment';
+
+export type URLParams = {
+  hazardLayerId?: string;
```

Comment:
If these are layer keys might as well type them as so? This will avoid unneeded casts down the line.

Reply:
You could also consider adding an assertion to ensure the values are indeed layer keys before storing them in the object